src/main/java/com/example/kolesnikov_advancedServer/repositories/UserRepo.java

```diff
@@ -0,0 +1,10 @@
+package com.example.kolesnikov_advancedServer.repositories;
+import com.example.kolesnikov_advancedServer.entity.UserEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public interface UserRepo extends JpaRepository<UserEntity, UUID> {
+    Optional<UserEntity> findByEmail(String email);
```

Comment:
```suggestion
    
    Optional<UserEntity> findByEmail(String email);
```
сразу для метода пропиши sql запрос